vm/src/builtins/object.rs

```diff
@@ -305,6 +305,21 @@ impl PyBaseObject {
     fn hash(zelf: PyObjectRef, vm: &VirtualMachine) -> PyResult<PyHash> {
         Self::slot_hash(&zelf, vm)
     }
+
+    #[pymethod(magic)]
+    fn sizeof(zelf: PyObjectRef) -> usize {
+        let mut size: usize = 0;
+        let isize = zelf.class().slots.itemsize;
+        if isize > 0 {
+            // TODO: CPython implementation has `PyVarObject.ob_size` field for each objects.
+            // Its default value seems 0 but it is changed by dynamically.
+            let object_size = 0;
+            size = object_size * isize;
```

Comment:
About these comments, I couldn't find an API to get `PyObject`'s size yet. 😞 

You can see the original implementation at https://github.com/python/cpython/blob/6d430ef5ab62158a200b94dff31b89524a9576bb/Objects/typeobject.c#L5481-L5494.

Reply:
```c
PyTypeObject PyBaseObject_Type = {
    PyVarObject_HEAD_INIT(&PyType_Type, 0)
    "object",        
...
```

I think that second argument in `PyVarObject_HEAD_INIT` is `ob_size` and that `PyVarObject_HEAD_INIT` assigned to `ob_base` in each types

Reply:
Thank you! It was helpful to analyze this issue again. 🙏🏻